tests/checkpointing/test_checkpoint_callback_frequency.py

```diff
@@ -102,3 +103,42 @@ def training_step(self, batch, batch_idx):
 
     # make sure types are correct
     assert save_mock.call_count == expected
+
+
+@mock.patch('torch.save')
+@RunIf(special=True, min_gpus=2)
+@pytest.mark.parametrize(['k', 'epochs', 'val_check_interval', 'expected'], [(1, 1, 1.0, 1), (2, 2, 0.3, 5)])
+def test_top_k_ddp(save_mock, tmpdir, k, epochs, val_check_interval, expected):
```

Comment:
something is wrong with this test, shouldn't it fail with an error saying that the monitor key is not found?
`self.log uses "my_loss" but monitor="my_loss_step"`

Reply:
nvm! 
it logs both on step and epoch, therefore a _step postfix is added